test/1_stdlib/DispatchDeprecationMacOS.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %swift -parse -target x86_64-apple-macosx10.9 -verify -disable-objc-attr-requires-foundation-module -sdk %sdk %s
```

Comment:
Do you actually need `-disable-objc-attr-requires-foundation-module`?  (and in the other test, too.)